<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="participant_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_participant_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_participant_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secret_santa_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>SECRET SANTA! </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>451</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Participant Info</string>
    </property>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>49</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_edit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="address_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Adderss_edit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>191</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ideas_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ideas</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ideas_edit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>159</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="exclude_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_exclude_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_exclude_button">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>460</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="exclude_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exclude</string>
     </property>
    </widget>
    <widget class="QComboBox" name="exclude_comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>460</y>
       <width>241</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="participant_list_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Participant List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionImport"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
